README.md

```diff
@@ -54,7 +54,7 @@ The table below outlines the payload structure for various transaction flows in
 |               | on_status_picked           | on_status_pending          |            | on_status_picked           | on_status_pending             |
 |               | on_status_out_for_delivery | on_status_packed           |            | on_status_out_for_delivery | on_status_packed              |
 |               | on_status_delivered        | on_status_picked           |            | on_cancel                  | on_status_picked              |
-|               |                            | on_status_out_for_delivery |            |                            | on_status_out_for_delivery    |
+|               |                            | on_status_out_for_delivery |            | on_status_rto_delivered    | on_status_out_for_delivery    |
```

Comment:
RTO can be disposed as well. So both the options should be provided; on_status (RTO-Delivered) and on_status (RTO-Disposed)